### PR TITLE
Fixing bugs

### PR DESCRIPTION
**Pull request by manassharma158**
- Clear Whiteboard bug fixed. 
- User-level handling bug fixed.

### CODE REVIEW

**Comment by manassharma158** [comment]
@bruise-88 The WhiteBoardStateManager interface was updated. Please fix your test accordingly. That's why the build error for Testing project.

**Comment by manassharma158** [comment]
@Yogesh7920 Can you merge this ? Whiteboard-UX will need this to raise a PR.

**Comment by bruise-88** [comment]
Can you please share your interface file?

**Comment by manassharma158** [comment]
> Can you please share your interface file?

IClientBoardStateManager. 
Update in the interface is in this PR.

File: Whiteboard/Client/Interfaces/IClientBoardStateManager.cs [hunk 1]
```diff
@@ -56,6 +56,7 @@ public interface IClientBoardStateManager
         /// <summary>
         /// Clears the whiteboard to fresh.
         /// </summary>
-        void ClearWhiteBoard();
+        /// <returns>Boolean to freeze whiteboard or not.</returns>
+        bool ClearWhiteBoard();
```
Comment (manassharma158) [inline Whiteboard/Client/Interfaces/IClientBoardStateManager.cs:60]: Here's the change @bruise-88 .

Comment (bruise-88) [reply Whiteboard/Client/Interfaces/IClientBoardStateManager.cs:60]: Give me 10 mins I will fix this and launch a PR

**Pull request merged**